### DISCUSSION TITLE
The package java.util is accessible from more than one module: <unnamed>, java.base

### DISCUSSION THREAD

**Discussion by guw** [eclipse-jdt/eclipse.jdt.core#2421] [Q&A] [answered]
Anyone knows how to fix:
```
The package java.util is accessible from more than one module: <unnamed>, java.base
```

The project setup looks ok (JDK 17). Could there be a jar leaking in `java.util` and therefor confusing Eclipse compiler? Or is it related to project settings?

**Comment by iloveeclipse** [comment, marked as answer]
Please check https://stackoverflow.com/questions/55571046/eclipse-is-confused-by-imports-accessible-from-more-than-one-module

**Reply by guw** [reply]
Thanks Andrey! There was indeed an org.osgi.foundation jar in the classpath causing the problem.
